src/include/binder/binder.h

```diff
@@ -15,9 +16,13 @@ struct CreateTableInfo;
 struct BaseScanSource;
 } // namespace parser
 
+namespace catalog {
```

Comment:
Do we need this?

Reply:
ah yeah. I was thinking to remove the include of `catalog.h` (but didn't end up doing it). that's why added extra forward declarations here. removed `catalog.h` now.